docs/rule-updates.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: rule-updates
+description: >-
+  Rule updates document significant rule updates made monthly.
+toc_max_heading_level: 2
+---
+
+import LinkToRegistryRule from "/src/components/LinkToRegistryRule"
+
+# Rule updates
+
+Welcome to monthly rule updates! This documet includes selected new rules and updates about community and free tier rules.
```

Comment:
```suggestion
Welcome to monthly rule updates! This document includes selected new rules and updates about community and free tier rules.
```

Also, I find the phrase “community and free tier rules” confusing in part because our free tier is called the community tier.

Reply:
@pabloest true, should we say community and team tier rules? But I am not sure if we use this specific distinction here.

Reply:
```suggestion
Welcome to monthly rule updates! This document includes selected new rules and rule updates made by the community and r2c.
```